tests/fv3_conf/rrfs_warm_run.IN

```diff
@@ -0,0 +1,30 @@
+rm -fr INPUT RESTART
+mkdir INPUT RESTART
+
+OPNREQ_TEST=${OPNREQ_TEST:-false}
+SUFFIX=${RT_SUFFIX}
+cp -r @[INPUTDATA_ROOT]/FV3_input_data_conus13km/INPUT/* INPUT/
+
+for x in global_glacier.2x2.grb global_h2oprdlos.f77 global_maxice.2x2.grb \
+         global_o3prdlos.f77 global_snoclim.1.875.grb global_zorclim.1x1.grb \
+         CFSR.SEAICE.1982.2012.monthly.clim.grb RTGSST.1982.2012.monthly.clim.grb ; do
+  cp -f @[INPUTDATA_ROOT]/FV3_fix/"$x" .
+done
+
+cp    @[INPUTDATA_ROOT]/FV3_input_data_conus13km/global_soilmgldas.t126.384.190.grb .
+cp    @[INPUTDATA_ROOT]/FV3_input_data_conus13km/seaice_newland.grb .
+cp    @[INPUTDATA_ROOT]/FV3_input_data_conus13km/Thompson_MP_MONTHLY_CLIMO.nc .
```

Comment:
I am reproducing the exact input directory of the test case I was given. Now is not the time for unnecessary changes.

Reply:
@SamuelTrahanNOAA We tried very hard to unify the fix files to avoid confusion related to fix files (same name with different content used in different tests, etc). So far, all the non-resolution related atm fixed files that are same version as those fixed files Kate is maintaining are in INPUT_DATA_ROOT/FV3_fixed. The resolution related lat/lon grid fixed files are in INPUT_DATA_ROOT/FV3_input_data*, the resolution related native tile grid fixed files are in INPUT_DATA_ROOT/FV3_fixed_tiled. New fixed files required by certain feature tests are in their own data input directory. I know there might still be some fixed data that are not in correct place yet, but we'd like to get them corrected soon. From what I see, we will copy the seaice_newland.grb to FV3_fixed. Thompson_MP_MONTHLY_CLIMO.nc is not used, the global_soilmgldas.t126.384.190.grb is in FV3_input_data. So we will copy FV3_input_data_conus13km with INPUT data directory. Please let me know if there is any issue.

Reply:
Testing this now.

Reply:
Fix files are no longer in the FV3_input_data_conus13km directory.